package.json

```
@@ -52,6 +55,12 @@
     "typescript": "5.4.5",
     "vite": "^5.2.11"
   },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "prettier --write",
+      "eslint --fix"
+    ]
+  },
```

IMO, it's probably a good idea to not give the `eslint --fix` command as a default, as some people don't run eslint very often for performance and other reasons, and it's easy to extend with additional settings.

```suggestion
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": [
      "prettier --write"
    ]
  },
```